packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -710,6 +714,11 @@ class _IOSSimulatorLogReader extends DeviceLogReader {
       return null;
     }
 
+    // Starts with space(s) - continuation of the multiline message
+    if (RegExp(r'\s*').matchAsPrefix(string) != null && !_lastLineMatched) {
```

Comment:
shouldn't this be `RegExp(r'\s+')`?

Reply:
@christopherfujino 🤦‍♂ 
I was even thinking to type `+` because "one or more" but typed familiar `*` anyway. 

Fixed and added test for that.